poetry.lock

```diff
@@ -6363,8 +6358,8 @@ tqdm = "*"
 [package.source]
 type = "git"
 url = "https://github.com/OpenDevin/SWE-bench.git"
-reference = "HEAD"
-resolved_reference = "7b0c4b1c249ed4b4600a5bba8afb916d543e034a"
+reference = "xw/attempt-fix-django-parsing"
+resolved_reference = "c61bc104ae5113dd58c228004e7b52293ed73bbd"
```

Comment:
We may want to remember to remove this when it's fixed in swe-bench

Reply:
Definitely! When my other PR on SWE-Bench repo is merged, i will update this.

Reply:
Just popping in to indicate the SWE-bench PR @xingyaoww posted was merged! Love the work OpenDevin is doing 😄

Reply:
Actually, maybe I'll make a PR for this 😛